packages/astro/src/core/sync/index.ts

```diff
@@ -62,6 +63,7 @@ export default async function sync(
 		settings,
 		logger,
 	});
+	createRouteManifest({ settings, fsMod: fs }, logger);
```

Comment:
This is unfortunate, but otherwise config:done's build output would be inaccurate. In the future we might use this for route type gen anyway, so perhaps not too bad.